src/app/Scenes/MyCollection/Screens/CollectedArtistsPrivacy/MyCollectionCollectedArtistsPrivacyArtistsList.tsx

```diff
@@ -0,0 +1,108 @@
+import { Flex, Spinner, Text, useSpace } from "@artsy/palette-mobile"
+import { MyCollectionCollectedArtistsPrivacyArtistsList_me$key } from "__generated__/MyCollectionCollectedArtistsPrivacyArtistsList_me.graphql"
+import { SelectArtistToShareListItem } from "app/Scenes/MyCollection/Components/SelectArtistToShareListItem"
+import { FlatList } from "react-native"
+import { usePaginationFragment } from "react-relay"
+import { graphql } from "relay-runtime"
+
+interface MyCollectionCollectedArtistsPrivacyArtistsListProps {
+  me: MyCollectionCollectedArtistsPrivacyArtistsList_me$key
+}
+
+export const ARTIST_CIRCLE_DIAMETER = 100
+
+export const MyCollectionCollectedArtistsPrivacyArtistsList: React.FC<
+  MyCollectionCollectedArtistsPrivacyArtistsListProps
+> = ({ me }) => {
+  const space = useSpace()
+
+  const { data, hasNext, loadNext, isLoadingNext } = usePaginationFragment(
+    myCollectionCollectedArtistsPrivacyArtistsListPaginationFragment,
+    me
+  )
+
+  const handleLoadMore = () => {
+    if (!hasNext || isLoadingNext) {
+      return
+    }
+
+    loadNext(10)
+  }
+
+  const userInterests = data.userInterestsConnection?.edges || []
+
+  if (userInterests.length === 0) {
+    return null
+  }
+
+  return (
+    <FlatList
+      showsHorizontalScrollIndicator
+      data={userInterests}
+      renderItem={({ item }) => {
+        if (item?.internalID && item.node) {
+          return (
+            <SelectArtistToShareListItem
+              key={item?.internalID}
+              artist={item?.node!}
+              interestID={item?.internalID}
+              private={item.private}
+            />
+          )
+        }
+        return null
+      }}
+      contentContainerStyle={{ paddingBottom: space(6) }}
+      ListHeaderComponent={HeaderComponent}
+      ListFooterComponent={() => {
+        if (!!isLoadingNext) {
```

Comment:
```suggestion
        if (isLoadingNext) {
```

should work as well or an [early return](https://medium.com/swlh/return-early-pattern-3d18a41bba8) 🤔

Reply:
Good point. I copied another component and didn't look into it thoroughly😄